### PR TITLE
tutorial(functions): update intro to guide more users

### PR DESCRIPTION
**Pull request by lucasscw**
### Your checklist for this pull request

- [ ] Check that the commit messages match our requested structure.
- [ ] Name your pull request according to the [contribution guidelines](https://github.com/scaleway/docs-content/blob/main/docs/CONTRIBUTING.md).

### Description
Please describe your pull request.

### CODE REVIEW

File: tutorials/transform-images-using-faas-and-nodejs/index.mdx [hunk 1]
```diff
@@ -33,18 +33,29 @@ This project contains two functions where:
...
-## Creating the BucketScan function
+## Project Set Up (For Serverless Framework)
 
-1. Set the following environment variables at the function level:
+1. Using the `serverless` command import the project from our Github Example Repository:
+  ```bash
+  serverless create --path nodetransform --template-url https://github.com/scaleway/serverless-examples/tree/main/functions/image-transform-node
+  ```
+2. Install the npm module using
+  ```npm install```
```
[the remark this answers is not in the document]
Comment (lucasscw) [reply tutorials/transform-images-using-faas-and-nodejs/index.mdx:46]: The code is only on one line isn't it?

**Review by bene2k1 — APPROVED** [review]

**Review by ldecarvalho-doc — APPROVED** [review]

File: tutorials/transform-images-using-faas-and-nodejs/index.mdx [hunk 2]
```diff
@@ -33,18 +33,29 @@ This project contains two functions where:
...
   - You have generated your [Scaleway API keys](/identity-and-access-management/iam/how-to/create-api-keys/)
-  - **(Optional)** You have installed the [Serverless Framework](https://serverless.com/) with Scaleway's Serverless Plugin
+  - **(Recommended)** You have installed the [Serverless Framework](https://serverless.com/) with [Scaleway's Serverless Plugin](https://github.com/scaleway/serverless-scaleway-functions)
 </Message>
 
-## Creating the BucketScan function
+## Project set up (for Serverless Framework)
 
-1. Set the following environment variables at the function level:
+1. Using the `serverless` command import the project from our Github Example Repository:
```
Comment (nedaehm) [inline tutorials/transform-images-using-faas-and-nodejs/index.mdx:41]:
```suggestion
1. Using the `serverless` command import the project from our Github example repository:
```

File: tutorials/transform-images-using-faas-and-nodejs/index.mdx [hunk 3]
```diff
@@ -33,18 +33,29 @@ This project contains two functions where:
...
+  serverless create --path nodetransform --template-url https://github.com/scaleway/serverless-examples/tree/main/functions/image-transform-node
+  ```
+2. Install the npm module using
+  ```npm install```
+3. Fill the `serverless.yml` file with your Scaleway credentials
+4. Set the following environment variables at the namespace level (provider section in `serverless.yml`):
     * `SOURCE_BUCKET`: Name of your source bucket
     * `S3_ENDPOINT_URL`: Scaleway’s Object Storage URL (`http://s3.{region}.scw.cloud`)
     * `SECRET_KEY_ID`: Your Scaleway API key ID
     * `SECRET_KEY:` Your Scaleway API key
```
Comment (nedaehm) [inline tutorials/transform-images-using-faas-and-nodejs/index.mdx:52]:
```suggestion
    * `SECRET_KEY:` Your Scaleway secret key
```

File: tutorials/transform-images-using-faas-and-nodejs/index.mdx [hunk 4]
```diff
@@ -269,4 +275,4 @@ The following steps are adapted from the [AWS lambda](https://docs.aws.amazon.co
 
 Please refer to [ways to deploy functions](/serverless/functions/reference-content/deploy-function) documentation to deploy this project.
```
Comment (nedaehm) [inline tutorials/transform-images-using-faas-and-nodejs/index.mdx:276]:
```suggestion
Read our documentation for more information on [ways to deploy functions](/serverless/functions/reference-content/deploy-function) to deploy this project. 
```

File: tutorials/transform-images-using-faas-and-nodejs/index.mdx [hunk 5]
```diff
@@ -269,4 +275,4 @@ The following steps are adapted from the [AWS lambda](https://docs.aws.amazon.co
 
 Please refer to [ways to deploy functions](/serverless/functions/reference-content/deploy-function) documentation to deploy this project. 
 
-For the full project + deployment using Serverless Framework refer to our [serverless-examples project](https://github.com/scaleway/serverless-examples/tree/main/functions/image-transform-node)
\ No newline at end of file
+For the full project and deployment using Serverless Framework refer to our [serverless-examples project](https://github.com/scaleway/serverless-examples/tree/main/functions/image-transform-node)
```
Comment (nedaehm) [inline tutorials/transform-images-using-faas-and-nodejs/index.mdx:278]:
```suggestion
For the full project and deployment using Serverless framework, refer to our [serverless-examples project](https://github.com/scaleway/serverless-examples/tree/main/functions/image-transform-node)
```

Comment (ofranc) [reply tutorials/transform-images-using-faas-and-nodejs/index.mdx:278]: plus haut on a pas mis de majuscule @nedaehm du coup tu veux faire quoi ?

Comment (nedaehm) [reply tutorials/transform-images-using-faas-and-nodejs/index.mdx:278]: On peut laisser avec une maj alors

**Review by thomas-tacquet — APPROVED** [review]

**Pull request merged**